Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -32,7 +32,7 @@ In this case, the most flexible solution is to use the primary focus handler as
 > [!IMPORTANT]
 > Please note that if hand rays are enabled, the head or eye gaze focus pointer are disabled as soon as the hands come into view. If you want to support a [_'look and pinch'_ interaction, you need to disable the hand ray](EyeTracking_EyesAndHands.md#how-to-disable-the-hand-ray). In our eye tracking sample scenes, we have disabled the hand ray to allow for showcasing richer interactions using eyes + hand motions - see for example [Eye-Supported Positioning](EyeTracking_Positioning.md).
 
-**2. Use both eye focus and hand rays at the same time:** 
+[**2. Use both eye focus and hand rays at the same time:**](EyeTracking_TargetSelection.html#2-independent-eye-gaze-specific-eyetrackingtarget)
```

Comment:
>EyeTracking_TargetSelection.htm [](start = 60, length = 31)

Since this is within the same file, I *think* you can omit the EyeTracking_TargetSelection.html part of it.

Reply:
Oh ok, good to know! Updated! :)